ci/jjb_service_job.yml

```diff
@@ -0,0 +1,22 @@
+- job:
+    name: 'centos-container-pipeline-service-job-openshift'
+    description: |
+        Managed by Jenkins Job Builder, do not edit manually!
+    node: container
+    properties:
+        - github:
+            url: https://github.com/CentOS/container-pipeline-service
+    triggers:
+        - github
+
+    wrappers:
+        - ansicolor
+    scm:
+        - git:
+            url: https://github.com/CentOS/container-pipeline-service.git
+            skip-tag: True
+            branches:
+                - openshift
+    builders:
+        - shell: |
+            jenkins-jobs --ignore-cache --conf ~/jenkins_jobs.ini update ci/job.yml
```

Comment:
There's no need to use absolute path here, right?

Reply:
I would say we are safer using absolute paths than relative paths, unless they are going to change constantly